PopulateTools/gobierto
# Gobierto Open Data: People exports

Connects to #303

### What does this PR do?

Adds an open data page at /datos with the links to download data in csv and json formats for the GobiertoPeople module, and also an RSS feed for blog posts.
The page is linked form the people sidebar.

### How should this be manually tested?

Check that the page exists at /datos and that you can download csv and json data and see the RSS feed
